src/explore-education-statistics-frontend/src/modules/find-statistics/components/FilterTag.module.scss

```diff
@@ -0,0 +1,24 @@
+@import '~govuk-frontend/govuk/base';
+
+.tag {
+  background: govuk-colour('light-grey');
+  border: 1px solid govuk-colour('mid-grey');
+  border-radius: 6px;
+  box-shadow: none;
+  line-height: 1;
+  margin: 0 govuk-spacing(1) 0 0;
+  padding: 8px;
+
+  &::before {
+    content: '✕';
```

Comment:
This kind of content can still show up for a screen reader. Would suggest we put this directly into a `span` element that's aria-hidden.

```tsx
<span aria-hidden>✕</span>
```